src/webapp/src/pages/Torrents.tsx

```diff
@@ -1,70 +1,60 @@
-import { Box, CircularProgress, CircularProgressLabel, Flex, Icon, IconButton, Input, Menu, MenuButton, MenuGroup, MenuItem, MenuList, Table, Tbody, Td, Th, Thead, Tr, useDisclosure } from "@chakra-ui/react";
+import "../scss/pages/torrents.scss";
+
 import React, { useState } from "react";
-import { MdCheck, MdDriveFileMove, MdMenu, MdOutlineFolder, MdOutlineRemove, MdPause, MdPlayArrow } from "react-icons/md";
-import { TbUpload } from "react-icons/tb";
+import { useTranslation, Trans } from "react-i18next";
+import {
+  Box, Center, CircularProgress, CircularProgressLabel, Flex,
+  HStack, Icon, IconButton, Input, Link, InputLeftElement,
+  InputGroup, Menu, MenuButton, MenuGroup, MenuItem, MenuList, Text,
+  Table, Tbody, Td, Th, Thead, Tr,
+  useColorMode, useDisclosure,
+} from "@chakra-ui/react";
+import {
+  BsArrowDown, BsArrowUp, BsCheck2, BsFilter, BsFolder, BsFolderFill,
+  BsList, BsPause, BsPauseFill, BsPlayFill, BsQuestion, BsTable,
+  BsThreeDotsVertical, BsTrash2Fill, BsX
+} from "react-icons/bs";
+import {
+  getState,
+  getStateColor,
+  isPaused,
+} from "../utils/torrentStates";
 import { trpc } from "../utils/trpc";
 import filesize from "filesize";
+import Loading from "../components/Loading";
 import { MoveTorrentModal } from "../components/MoveTorrentModal";
 import { RemoveTorrentDialog } from "../components/RemoveTorrentDialog";
 
-function getColorFromState(state: number): string {
-  switch (state) {
-    case 3: return "blue.400";
-    case 5: return "green.400";
-  }
-  return "black";
-}
+function ProgressLabel({ ...props }: any) {
+  const { t, torrent, showNumber } = props;
+  let state: string = getState(torrent);
+  let color: string = getStateColor(state);
 
-function isCompleted(torrent: {state: number, flags: number}): boolean {
-  return torrent.state === 5 && isPaused(torrent.flags);
-}
+  const icons = {
```

Comment:
I've tried to use the `Md` (material design) icons - see if you can find anything fitting in those